### ISSUE TITLE
[bug] Import sample doesn't work properly

### ISSUE THREAD

**Issue by mittalyashu** [TablePlus/diagram-plugin#12]
There was some existing data in one of the page _(IDK what you guys call it, page, maybe)_ and when I try the import sample feature.

I remove all of the existing data and didn't added any sample data on top of it. Well, in the first place I think it should not replace or overwrite the data, and create a new page.

![CleanShot 2020-10-23 at 09 33 34](https://user-images.githubusercontent.com/29014463/96955458-3f919700-1513-11eb-9731-87f982540084.gif)

**Comment by dvkndn** [comment]
> remove all of the existing data [...] Well, in the first place I think it should not replace or overwrite the data, and create a new page.

Well this is the expected behavior :( This is why we called it "Import" instead of "Open". I think it's a common naming.

Anyway, we will try to think of a way to support both if possible

> didn't added any sample data on top of it

This, however, is indeed a bug. We will need to take a look at it. Unfortunately the current "Report issue" feature is being buggy so I can't ask you for a sample data :(

**Comment by mittalyashu** [comment]
> This is why we called it "Import" instead of "Open"

In that case, should it just append the data instead of replacing the data, there can be cases where users might have not even saved their changes.

---

Just a general question, why don't you guys built the plugin in native language just like the application itself?

**Comment by dvkndn** [comment]
> In that case, should it just append the data instead of replacing the data, there can be cases where users might have not even saved their changes.

Yeah we already have that option (but hide it). Let us see turning it on :+1: 

> Just a general question, why don't you guys built the plugin in native language just like the application itself?

There are several reasons but the one I can share is that we'd love to build an open, cross-platform plugin system (the plugins are used in TablePlus Mac and Windows without any modification, with Linux coming soon).

Another important reason is that for some specific jobs like this, we believe the web platform is mature enough.
